docs/operators/csvParserOperator.md

```diff
@@ -0,0 +1,79 @@
+csv Parser Operator
+=============
+
+## Operator Objective
+This operator is designed to parse csv records and construct a map or Plain Old Java Object ("POJO") out of it. User need to provide the schema to describe the csv data. Based on schema definition the operator will parse the incoming record to object map and POJO.  User can also provide constraints if any, in the schema. The incoming record will be validated against those constraints. Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message.
```

Comment:
1. Let's use the word delimited instead of CSV. 
2. Having a section describing all the constraints that we currently support would be helpful.